New prestashop v4.2.0 changelog (#882)
mercadopago/devsite-docs: guides/plugins/official/shopify.en.md

# Mercado Pago for Shopify

## Introduction

Mercado Pago solutions in Shopify will allow you to collect online on your website, with no need to have technical or programming knowledge.

> Mercado Pago is an official Shopify partner: we provide security to all payments made in your store.

### Integration types

----[mlb]----
Choose the solution that best suits your business:

__Mercado Pago Transparent Checkout__: Shopify *app extension* that provides a fast and secure shopping experience, without leaving your store! You can install it from [Shopify app store](https://apps.shopify.com/).
------------

> __Mercado Pago Redirect Checkout__: configure Mercado Pago Checkout, so that your customers finalize their payment on our website. It is already pre-installed in your Shopify administrator.

Reviewer: ```suggestion
__Mercado Pago Redirect Checkout__: configure Checkout Pro, so that your customers finalize their payment on our website. It is already pre-installed in your Shopify administrator.
```

| Mobile experience  | Provides a thought and optimized purchase flow for mobile devices.                             |
| Shipments          | Connect the checkout with your shipping tool and offer deliveries during the purchase process. |
| Ads tracking       | Track your marketing campaigns in detail throughout the process.                               |
| Payment methods    | Accept payments with credit and debit card, cash and money in Mercado Pago account.            |
| Financing          | Sell in installments and offer the promotions available.                                       |
| Payment as guest   | Allow your customers to pay even if they don’t have a Mercado Pago account.                    |

### How to install the Transparent Checkout?

Install Mercado Pago for Shopify automatically by following these steps from [Shopify app store](https://apps.shopify.com/):

1. Enter the website [https://apps.shopify.com/](https://apps.shopify.com/) from your browser and log in to your account. You can also access it from the Apps section of your Shopify panel.

> 1. Search for *“Mercado Pago Transparent Checkout”* in the apps offer.

Reviewer: ```suggestion
1. Search for “Mercado Pago Transparent Checkout” in the apps offer.
```

> 1. Click on *Add app*.

Reviewer: ```suggestion
1. Click on "Add app".
```

**Done! With this configuration you can start selling and get the payments you receive in your Mercado Pago account.**

> WARNING
>
> Importante
>
> The [credentials](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/faqs/credentials) are the keys that uniquely identify you within Mercado Pago. They will allow you to simulate payments in a test environment and receive real payments once you are ready to go to production.

### How to test the checkout?

The Mercado Pago app extension comes with the default testing environment, so that you can simulate payments in the store and verify that everything works well before you start receiving real payments from your customers.
 
This is where the test credentials you entered in the module when integrating Mercado Pago to your store come into play.

When you test, check that the payment flow works correctly.

**When you see that everything is going well, deactivate the Tests mode and go to the Production mode to receive real payments.**

### Go to production

To start receiving payments, you must [activate your credentials]([FAKER][CREDENTIALS][URL]/).

> > Check the [requirements to go to production](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/payments/api/goto-production/) if you have any questions about the process.

Reviewer: Creo que no es necesario sumar está referencia.

**Done! Mercado Pago Transparent Checkout is ready to receive payments online.**

------------

## Mercado Pago Redirect Checkout

Configure our module to finalize the purchase on Mercado Pago site during the purchase process in your Shopify store.

### What can I do with Mercado Pago payment gateway?

| Characteristics        | Description                                                                              |
|---|---|
| Mercado Pago Interface | We take care of it! You don't have to worry about implementing and designing a checkout. |
| Ads tracking           | Track your marketing campaigns in detail throughout the process.                         |
| Payment methods        | Accept payments with credit and debit card, cash and money in Mercado Pago account.      |
| Financing              | Sell in installments and offer the promotions available.                                 |
| Payment as guest       | Allow your customers to pay even if they don’t have a Mercado Pago account.              |
| Discounts              | Use coupons to offer promotions to your customers.                                       |

### How to configure Mercado Pago module?

> Follow these steps to process payments with Mercado Pago Checkout:

Reviewer: ```suggestion
Follow these steps to process payments with Checkout Pro:
```

1. In Shopify admin panel, go to the *Payments* section under *Settings*.
1. In the *Third-party providers* box, search for Mercado Pago and select it.
1. Get the [credentials]([FAKER][CREDENTIALS][URL]) **Client id** e **Client secret**, and paste them into the corresponding fields.
1. Choose the payment methods you want to offer your clients.
1. Be sure to select Test mode to verify everything is working properly before going into production.
1. Activate the module to save the changes.

**Done! Mercado Pago was installed and configured. Once you have done tests, you can come to this same section to deactivate the test environment and receive real payments.**

> NOTE
>
> Nota
>

> > Check [Mercado Pago Checkout](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/payments/web-payment-checkout/introduction/) documentation to learn more about all its features and functionalities.

Reviewer: ```suggestion
> Check [Checkout Pro](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/payments/web-payment-checkout/introduction/) documentation to learn more about all its features and functionalities.
```
